clientd3d/d3drender.c

```diff
@@ -1090,20 +1058,18 @@ void D3DRenderBegin(room_type *room, Draw3DParams *params)
 		D3DCacheFlush(&gObjectCacheSystem, &gObjectPool, 1, D3DPT_TRIANGLESTRIP);
 
 		SetZBias(gpD3DDevice, ZBIAS_DEFAULT);
-		gpD3DDevice->SetRenderState(D3DRS_ZWRITEENABLE, FALSE);
-      
-		D3DRenderFramebufferTextureCreate(gpBackBufferTexFull, gpBackBufferTex[0],
-			gSmallTextureSize, gSmallTextureSize);
+		IDirect3DDevice9_SetRenderState(gpD3DDevice, D3DRS_ZWRITEENABLE, FALSE);
+
+		D3DRenderFramebufferTextureCreate(gpBackBufferTexFull, gpBackBufferTex[0], gSmallTextureSize, gSmallTextureSize);

```

Comment:
Your auto-formatter settings are kind of weird.  Why did it combine these two reasonably long lines into this one huge line?  We should try not to make the code worse....